src/components/bs5/footer/footerForgov.hbs

```diff
@@ -1,69 +1,93 @@
+                        </div>
+                        <div class="col-xs-12 col-md-6 col-lg-12">
+                            <a href="{{ contact.buttonLink }}" class="btn btn-outline-secondary">{{ contact.buttonLabel }}</a>
                         </div>
-                        {{/each}}
-
-                        <a href="{{ contact.buttonLink }}" class="btn btn-outline-secondary">{{ contact.buttonLabel }}</a>
-
                     </div>
-                
                 </div>
             </div>
 
-            <!-- Section 5: Social media and engagment -->
-            <div class="col-xs-12 col-md-6 col-lg-4 col-xl-3 border-column">
-
-                <div class="footer-column footer-contact">
-                
-                    <h2 class="footer-heading">{{ feedback.title }}</h2>
-
-                    <div class="footer-content">
-
-                        {{{ feedback.content }}}
+            {{!-- Section 2: Section links (Custom secondary nav items) --}}
+            <div class="col-xs-12 col-md-12 col-lg-3" aria-label="footer-navigation">
+                <div class="footer-column {{#if footerlinks.hasBorderColumn}}border-column{{/if}}">
+                    <nav class="nav footer-link-list flex-column">
+                        {{#each footerlinks.list}}
+                            <a class="nav-link" href="{{ link }}" target="{{ target }}">{{{ title }}}</a>
+                        {{/each }}
+                    </nav>
+                </div>
+            </div>
 
-                        <a href="{{ feedback.buttonLink }}" class="btn btn-outline-secondary">{{ feedback.buttonLabel }}</a>
+            {{!-- Section 3: Feedback form area --}}
+            {{#if feedbackFormEnabled}}
+            <div class="col-xs-12 col-md-12 col-lg-3">
+                <div class="footer-column {{#if feedbackForm.hasBorderColumn}}border-column{{/if}}">
+                    <div class="row">
+                        <div class="col-xs-12 col-md-6 col-lg-12">
+                            <h3 class="footer-heading">{{ feedbackForm.title }}</h3>
+                            <p>{{ feedbackForm.content }}</p>
+                        </div>
+                        <div class="col-xs-12 col-md-6 col-lg-12">
+                            <div id="qg-feedback-toggle" class="qg-footer-feedback-wrap">
+                                <a id="btn-footer-feedback" class="btn btn-global-secondary qg-feedback-toggle collapsed" data-bs-toggle="collapse" href="#qg-footer-feedback" role="button" aria-expanded="false" aria-controls="qg-footer-feedback" data-analytics-link-group="qg-feedback">
+                                    {{ feedbackForm.btnTitle }}
+                                </a>
+                                <div id="qg-footer-feedback" class="qg-footer-feedback__v2 collapse">
+                                    <script>
+                                        window.formioQldCdnVersion = "v2/v2.x.x-latest";
```

Comment:
ditto, move the inline script variable into script file is better, and let it use global if not overridden in by an earlier function.